### DISCUSSION TITLE
Mulighet for å slå av å vise type-logo på hver avgang

### DISCUSSION THREAD

**Discussion by forteller** [entur/tavla#564] [Ideas]
Hvis man har en boks som kun inneholder én type transport (kun buss feks), så kan det føles unødvendig og rotete å vise buss-logoen nedover på venstre side for hver eneste avgang. Det hadde vært fint med en innstilling for å slå av dette. Takk!

**Comment by lindtvedtsebastian** [comment]
This discussion is linked to the old (deprecated) version of Tavla and will therefore be closed.
